### PR TITLE
feat: Move schema endpoits to core API

### PR DESCRIPTION
**Pull request by x1m3**

### CODE REVIEW

**Review by martinsaporiti — APPROVED** [review]

File: internal/api/schemas.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+// ImportSchema is the UI endpoint to import schema metadata
+func (s *Server) ImportSchema(ctx context.Context, request ImportSchemaRequestObject) (ImportSchemaResponseObject, error) {
+	req := request.Body
+	if err := guardImportSchemaReq(req); err != nil {
+		log.Debug(ctx, "Importing schema bad request", "err", err, "req", req)
+		return ImportSchema400JSONResponse{N400JSONResponse{Message: fmt.Sprintf("bad request: %s", err.Error())}}, nil
+	}
+	iReq := ports.NewImportSchemaRequest(req.Url, req.SchemaType, req.Title, req.Version, req.Description)
+	schema, err := s.schemaService.ImportSchema(ctx, s.cfg.APIUI.IssuerDID, iReq)
```
Comment (x1m3) [inline internal/api/schemas.go:23]: This is wrong. s.cfg.APIUI.IssuerDID is dreprecated

File: internal/api/schemas.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	if err != nil {
+		log.Error(ctx, "Importing schema", "err", err, "req", req)
+		return ImportSchema500JSONResponse{N500JSONResponse{Message: err.Error()}}, nil
+	}
+	return ImportSchema201JSONResponse{Id: schema.ID.String()}, nil
+}
+
+// GetSchema is the UI endpoint that searches and schema by Id and returns it.
+func (s *Server) GetSchema(ctx context.Context, request GetSchemaRequestObject) (GetSchemaResponseObject, error) {
+	schema, err := s.schemaService.GetByID(ctx, s.cfg.APIUI.IssuerDID, request.Id)
```
Comment (x1m3) [inline internal/api/schemas.go:33]: This is wrong. s.cfg.APIUI.IssuerDID is dreprecated

File: internal/api/schemas.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	}
+	if err != nil {
+		log.Error(ctx, "loading schema", "err", err, "id", request.Id)
+	}
+	return GetSchema200JSONResponse(schemaResponse(schema)), nil
+}
+
+// GetSchemas returns the list of schemas that match the request.Params.Query filter. If param query is nil it will return all
+func (s *Server) GetSchemas(ctx context.Context, request GetSchemasRequestObject) (GetSchemasResponseObject, error) {
+	col, err := s.schemaService.GetAll(ctx, s.cfg.APIUI.IssuerDID, request.Params.Query)
```
Comment (x1m3) [inline internal/api/schemas.go:46]: This is wrong. s.cfg.APIUI.IssuerDID is dreprecated

**Review by x1m3 — COMMENTED** [review]
Do not use s.cfg.APIUI.IssuerDID. It is dreprecated

File: internal/api/schemas.go (hunk 1 above)
Comment (x1m3) [reply internal/api/schemas.go]: fixed

File: internal/api/schemas.go (hunk 3 above)
Comment (x1m3) [reply internal/api/schemas.go]: fixed

**Pull request merged**